.eslintrc.json

```diff
@@ -31,7 +31,8 @@
     {
       "files": [
         "./packages/d3fc-chart/src/**/*.js",
-        "./packages/d3fc-element/src/**/*.js"
+        "./packages/d3fc-element/src/**/*.js",
+        "./packages/d3fc-discontinuous-scale/src/**/*.js"
```

Comment:
Is this just to support `console.warn`? Maybe it's worth us adding a utility package to wrap this up so we don't need to add every package to this list?

Reply:
Is your main concern that by moving this package into this particular override section that it now permits all browser globals? How about we just enable console as a global variable across all packages?

https://github.com/d3fc/d3fc/pull/1631/files#diff-df39304d828831c44a2b9f38cd45289cR12

Reply:
It was and that definitely solves it in a nicer way. However, we now risk accidently leaving console logs in the code. I wonder if we should just excuse the individual log lines?

Reply:
We can prevent `console.log` explicitly:

https://github.com/d3fc/d3fc/pull/1631/files#diff-df39304d828831c44a2b9f38cd45289cR22